README.md

```diff
@@ -18,6 +18,9 @@
 [Installation](https://auto.gluon.ai/stable/install.html) | [Documentation](https://auto.gluon.ai/stable/index.html) | [Release Notes](https://auto.gluon.ai/stable/whats_new/index.html)
 
 AutoGluon automates machine learning tasks enabling you to easily achieve strong predictive performance in your applications.  With just a few lines of code, you can train and deploy high-accuracy machine learning and deep learning models on image, text, time series, and tabular data.
+
+AutoGluon was originally developed by researchers and engineers at AWS AI.
+
 </div>
```

Comment:
Just noticed this `</div>` should be moved to right before the first text line starting with "AutoGluon automates ...", we are unintentionally centering the text.